tests/path.rs

```diff
@@ -301,9 +300,8 @@ fn no_rebuild_dependency() {
     // This time we shouldn't compile bar
     assert_that(p.cargo("build"),
                 execs().with_stdout(""));
-    p.root().move_into_the_past();
+    p.root().move_into_the_future();
 
-    p.build(); // rebuild the files (rewriting them in the process)
```

Comment:
@alexcrichton I don't understand how this test is supposed to work. We move files into the past and immediately replace them...

Also, the assertion on the previous line looks suspicions, because we never print to stdout in build....